frontend/public/components/storage-class.tsx

```diff
@@ -19,26 +19,23 @@ export const isDefaultClass = (storageClass: K8sResourceKind) => {
 };
 
 const StorageClassHeader = props => <ListHeader>
-  <ColHead {...props} className="col-sm-4 col-xs-6" sortField="metadata.name">Name</ColHead>
-  <ColHead {...props} className="col-sm-4 col-xs-6" sortField="provisioner">Provisioner</ColHead>
+  <ColHead {...props} className="col-sm-5 col-xs-6" sortField="metadata.name">Name</ColHead>
+  <ColHead {...props} className="col-sm-5 col-xs-6" sortField="provisioner">Provisioner</ColHead>
   <ColHead {...props} className="col-sm-2 hidden-xs" sortField="reclaimPolicy">Reclaim <span className="hidden-sm">Policy</span></ColHead>
-  <ColHead {...props} className="col-sm-2 hidden-xs" sortField={`metadata.annotations['${defaultClassAnnotation}']`}>Default <span className="hidden-sm">Class</span></ColHead>
 </ListHeader>;
 
 const StorageClassRow: React.SFC<StorageClassRowProps> = ({obj}) => {
   return <div className="row co-resource-list__item">
-    <div className="col-sm-4 col-xs-6 co-break-word">
-      <ResourceLink kind={StorageClassReference} name={obj.metadata.name} namespace={undefined} title={obj.metadata.name} />
+    <div className="col-sm-5 col-xs-6 co-break-word">
+      <ResourceLink inline={true} kind={StorageClassReference} name={obj.metadata.name} namespace={undefined} title={obj.metadata.name} />
+      { isDefaultClass(obj).toString() ? <i><small className="text-muted"> &ndash; &nbsp; &nbsp; Default </small></i> : null }
```

Comment:
Visually this is still not right. See the comment here: https://github.com/openshift/console/pull/1332#discussion_r269276253

By the way, you can use `&&` here instead of a ternary.

```suggestion
      { isDefaultClass(obj).toString() && <i><small className="text-muted"> &ndash; &nbsp; &nbsp; Default </small></i> }
```

Reply:
Ack! Made the requested changes.